InventoryInStorePickupSalesAdminUi/Controller/Adminhtml/Order/NotifyPickup.php

```diff
@@ -78,8 +78,7 @@ public function execute(): ResultInterface
             return $this->resultRedirectFactory->create()->setPath('sales/*/');
         }
 
-        $result = $this->notifyOrderIsReadyForPickup->execute([(int)$order->getEntityId()]);
-
+        $result = $this->notifyOrderAreReadyForPickup->execute([(int)$order->getEntityId()]);
         if ($result->isSuccessful()) {
             if ($order->getEmailSent()) {
```

Comment:
What if email is sent asynchronously?

Reply:
This check does not make sense in case with asynchronous email sending, removed.